env/composite_env_wrapper.h

```diff
@@ -293,6 +293,10 @@ class CompositeEnvWrapper : public Env {
   // calls to env, and all file operations to fs
   explicit CompositeEnvWrapper(Env* env, FileSystem* fs)
       : env_target_(env), fs_env_target_(fs) {}
+  explicit CompositeEnvWrapper(Env* env, std::unique_ptr<FileSystem>&& fs)
+      : env_target_(env), fs_env_ptr_(std::move(fs)), fs_env_target_(fs_env_ptr_.get()) {}
```

Comment:
Can `CompositeEnvWrapper` keep `fs_env_ptr_` and remove `fs_env_target_`, since the latter is just `fs_env_ptr_.get()`?

Reply:
Yes, it can. Will make the change.